server/src/controllers/user/sendEmailController.ts

```diff
@@ -0,0 +1,23 @@
+import { NextFunction, Request, Response } from 'express';
+import { sendCode, sendEmail } from '../../utils';
+import customError from '../errors';
+
+const sendEmailController = async (req: Request, res: Response, next: NextFunction) => {
+  const { email } = req.body;
+  try {
+    await sendEmail.validateAsync({ email });
+
+    await sendCode(email);
+
+    res.status(200).json({
+      message: 'Email sent successfully',
+    });
+  } catch (err) {
+    if (err.name === 'ValidationError') {
```

Comment:
this errors we will handle it in express ERROR middleware,
what if we have an  error in twilio service? we will handle it also in error middle ware, Guess what the status Code will be? ;)

Reply:
I Think 503 Service is Unavailable,
I will do it